client/views/components/generic/date-utility/DateUtility.jsx

```diff
@@ -0,0 +1,61 @@
+import React, {PureComponent } from 'react';
+import { string, object  } from 'prop-types';
+import { BaseEntityQuery } from 'utils/genny';
+import distanceInWords from 'date-fns/distance_in_words';
+
+class DateUtility extends PureComponent {
+    static propTypes = {
+        attribute: string,
+        style: object,
+        field: string,
+        code: string,
+        formatType: string
+    }
+
+    getFromNowInWords = (code, attribute, field) => { 
+      /* if BECODE and ATTRIBUTE is supplied as props */
+      if(code && attribute){ 
+        const attributevalues = BaseEntityQuery.getBaseEntityAttribute(code, attribute);
+        const dateValue = attributevalues.valueDate;
```

Comment:
need a null check here, if the attribute doesnt exist, then the query will return null

Reply:
ok this has been resolved in latest commit pushed it